Formula/azure-cli.rb

```diff
@@ -26,23 +440,42 @@ def install
     ]
     components += Pathname.glob(buildpath/"src/command_modules/azure-cli-*/")
 
-    # Build source distributions
+    # Install dependencies
+    # note: Even if in 'resources', don't include 'futures' as not needed for Python3
+    # and causes import errors. See https://github.com/agronholm/pythonfutures/issues/41
+    deps = resources.map(&:name).to_set - ["futures"]
+    deps.each do |r|
+      resource(r).stage do
+        system "python3", *Language::Python.setup_install_args(libexec)
+      end
+    end
+
+    # Install CLI
     components.each do |item|
-      Dir.chdir(item) { system libexec/"bin/python", "setup.py", "sdist", "-d", buildpath/"dist" }
+      Dir.chdir(item) do
+        system "python3", *Language::Python.setup_install_args(libexec)
+      end
     end
 
-    # Install CLI using source distributions only
-    system libexec/"bin/pip", "install", "--no-binary", ":all:", "azure-cli", "-f", buildpath/"dist"
-    system libexec/"bin/pip", "install", "--no-binary", ":all:", "--force-reinstall", "-U", "azure-nspkg", "azure-mgmt-nspkg", "-f", buildpath/"dist"
+    # This replaces the `import pkg_resources` namespace imports from upstream
+    # with empty string as the import is slow and not needed in this environment.
+    File.open(site_packages/"azure/__init__.py", "w")
```

Comment:
```
touch site_packages/"azure/__init__.py"
```

Reply:
The files already exist and contain these contents:
```
import pkg_resources
pkg_resources.declare_namespace(__name__)
```

I want to empty the contents of those files.
`touch` would create the file if it didn't exist but doesn't empty its contents.